demos/projects/NXP/mimxrt1060/config/demo_config.h

```diff
@@ -13,7 +13,7 @@
  * This Model ID is tightly tied to the code implementation in `sample_azure_iot_pnp_simulated_device.c`
  * If you intend to test a different Model ID, please provide the implementation of the model on your application.
  */
-#define sampleazureiotMODEL_ID                                "dtmi:com:example:Thermostat;1"
+#define sampleazureiotMODEL_ID                                "dtmi:azure:iot:deviceUpdateModel;1"
```

Comment:
as a follow-up, should this be removed since we now set it somewhere else for adu?

Reply:
we can leave this since it works for the other samples. The ADU one will undef and redef the correct one.

Reply:
isn't Thermostat the one that works for the other samples?

Reply:
Yea we can maybe think about putting this model id in the samples themselves instead of the demo config. Since it's particular to the sample and not the whole build for a given platform.